core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -542,6 +560,8 @@ public GraphHopper init(GraphHopperConfig ghConfig) {
 
         // elevation
         this.smoothElevation = ghConfig.getBool("graph.elevation.smoothing", false);
+        this.longEdgeSamplingDistance = ghConfig.getDouble("graph.elevation.long_edge_sampling_distance", 0);
+        this.elevationSimplifyFactor = ghConfig.getDouble("graph.elevation.simplify_factor", 0);
```

Comment:
Without bilinear interpolation, the output can look pretty bad because it enhances the resolution on blockiness. Do we want to warn when long edge sampling is used without bilinear interpolation?

For example with SRTM and no bilinear interpolation:
![image](https://user-images.githubusercontent.com/1480504/76613437-7e2e9000-64f4-11ea-9cb1-bca484a1876e.png)